src/server_manager/ui_components/outline-server-view.html

```diff
@@ -758,31 +786,36 @@ <h3>[[localize('server-help-access-key-title')]]</h3>
       closeGetConnectedHelpBubble: function() {
         this.$.getConnectedHelpBubble.hide();
       },
-      _selectedTabChanged(selectedTab) {
-        if (this.selectedTab === 'settings') {
-          this.$.addAccessKeyHelpBubble.hide();
-          this.$.serverSettings.update(this.serverName, this.metricsEnabled);
-        }
+      closeDataLimitsHelpBubble: function() {
+        this.$.dataLimitsHelpBubble.hide();
+      },
+      showDataLimitsHelpBubble: function() {
+        this.showHelpBubbleOnce('dataLimitsHelpBubble', 'settingsTab', 'up', 'right');
       },
       // initHelpBubbles should be called after this outline-server-view
       // is on the screen (e.g. selected in iron-pages).  If help bubbles
       // are initialized before this point, setPosition will not work and
       // they will appear in the top left of the view.
       initHelpBubbles: function() {
-        this.showHelpBubbleOnce('addAccessKeyHelpBubble', 'addAccessKeyRow', 'down', 'left');
+        if (!this.showHelpBubbleOnce('addAccessKeyHelpBubble', 'addAccessKeyRow', 'down', 'left') &&
+            this.canSetAccessKeyDataLimit) {
+          // Show the data limits help bubble if the access key one has been already shown.
+          this.showDataLimitsHelpBubble();
+        }
       },
       showHelpBubbleOnce: function(helpBubbleId, positionTargetId,
```

Comment:
UI components should be dumb and shouldn't have access to storage. Could you move the "once" logic to app.ts instead?

And the `showDataLimitsHelpBubble()` should always show the bubble.

Reply:
Done in 689d8b9.